unreleased_history/bug_fixes/compressed_secondary_cache_account.md

```diff
@@ -0,0 +1 @@
+Fix under counting of allocated memory in the compressed secondary cache due to looking at actual block size rather than usable memory.
```

Comment:
The way this is stated, it's not completely obvious whether "looking at actual block size" is bad/old or new/good.